samples/40.timex-resolution/src/main/java/com/microsoft/bot/sample/timex/resolution/Parsing.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.timex.resolution;
+
+import com.microsoft.recognizers.datatypes.timex.expression.Constants.TimexTypes;
+import com.microsoft.recognizers.datatypes.timex.expression.TimexProperty;
+
+   /** The TimexProperty class takes a TIMEX expression as a string argument in its constructor.
+    * This pulls all the component parts of the expression into properties on this object. You can
+    * then manipulate the TIMEX expression via those properties.
+    * The "Types" property infers a datetimeV2 type from the underlying set of properties.
+    * If you take a TIMEX with date components and add time components you add the
+    * inferred type datetime (its still a date).
+    * Logic can be written against the inferred type, perhaps to have the bot ask the user for disambiguation.
+    */
+public class Parsing
+{
+    private static void describe(TimexProperty t)
+    {
+        System.out.println(t.getTimexValue());
```

Comment:
do we need to add a [space](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/40.timex-resolution/Parsing.cs#L22) at the end here? Wondering since [JS](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/javascript_nodejs/40.timex-resolution/parsing.js#L14) doesn't have it, so I'm not sure.